src/server/routes/github.js

```diff
@@ -18,6 +19,9 @@ router.post('/github/webhook', createWebhook);
 router.use('/github/user', json());
 router.get('/github/user', getUser);
 
+router.use('/github/orgs', json());
```

Comment:
It's just a GET, so I'm not sure if we need it, but it does clear the cache, so it wouldn't hurt to verify.